src/index.ts

```diff
@@ -199,7 +199,7 @@ const graphqlServer = graphqlHTTP((req, res, params) => {
     }),
     validationRules,
     extensions: ({ document, result }) =>
-      createExtensions(document, result, requestID),
+      createExtensions(document, result, requestID, userAgent),
```

Comment:
This adds some more useful debug information to GraphiQL output

Reply:
<img width="425" alt="Screenshot 2022-11-17 at 4 14 10 PM" src="https://user-images.githubusercontent.com/236943/202587267-e045f8a9-bffe-4c4e-930b-52e05b5c8f2f.png">